src/panels/config/automation/condition/types/ha-automation-condition-trigger.ts

```diff
@@ -70,8 +70,11 @@ export class HaTriggerCondition extends LitElement {
   }
 
   private _automationUpdated(config?: AutomationConfig) {
+    const seenIds = new Set();
     this._triggers = config?.trigger
-      ? ensureArray(config.trigger).filter((t) => t.id)
+      ? ensureArray(config.trigger)
+          .filter((t) => t.id)
+          .filter((t) => (seenIds.has(t.id) ? false : seenIds.add(t.id)))
```

Comment:
Combine this in the same filter so we don't go over the array twice

Reply:
Fixed.